daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/EnvironmentInterface.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.daml.lf
+package iface
+
+import data.Ref.Identifier
+
+import scala.collection.breakOut
+import scala.collection.immutable.Map
+import scalaz.syntax.std.map._
+import scalaz.Semigroup
+
+/** The combination of multiple [[Interface]]s, such as from a dar. */
+final case class EnvironmentInterface(typeDecls: Map[Identifier, InterfaceType])
```

Comment:
I never liked the name... can we call it `InterfaceUnion`. That is exactly what the combination of multiple `Interface`'s is, right? Or does environment mean anything in this context?

Reply:
"Environment" here means the environment of types available to your program, symbolized `Γ ` in the spec (and most specs for typed languages).

"Union" implies something you cannot do; you cannot union two `Interface`s, or two dalfs for that matter. Do you have a different name suggestion?

Reply:
By the way, the other common pronunciation of `Γ` is "context", which I think is clearly not as good as "environment" in this, er, context. cc @remyhaemmerle-da